MDANSE_GUI/Src/MDANSE_GUI/MolecularViewer/Controls.py

```diff
@@ -264,7 +267,22 @@ def createSidePanel(self):
         layout.addWidget(wrapper5)
         # the database of atom types
         # self._database = TrajectoryAtomData()
-        self.layout().addWidget(base, 0, 2, 2, 1)  # row, column, rowSpan, columnSpan
+        self.layout().addWidget(
+            absolute_base, 0, 2, 2, 1
```

Comment:
```suggestion
            absolute_base, 
            row=0, 
            column=2, 
            rowSpan=2, 
            columnSpan=1,
```
Then no need for comment.

Reply:
On my platform, the modified code triggers:
```
TypeError: arguments did not match any overloaded call:
  addWidget(self, w: Optional[QWidget]): 'row' is not a valid keyword argument
  addWidget(self, a0: Optional[QWidget], row: int, column: int, alignment: Qt.AlignmentFlag = Qt.Alignment()): 'row' is not a valid keyword argument
  addWidget(self, a0: Optional[QWidget], row: int, column: int, rowSpan: int, columnSpan: int, alignment: Qt.AlignmentFlag = Qt.Alignment()): 'row' is not a valid keyword argument
```

Reply:
Huh. Maybe it's positional only, guess that's part of the C interface. Nevermind then.